.github/workflows/ios-maestro.yml

```
@@ -100,46 +120,251 @@ jobs:
+                target: [bare, expo]
+                shard-index: [0, 1]
+        env:
+            MAESTRO_DEBUG_OUTPUT_DIRECTORY: ${{ github.workspace }}/packages/react-native-payments-example/artifacts/maestro-ios-${{ matrix.target }}-${{ matrix.shard-index }}
+
+        steps:
+            -   name: Checkout repository
+                uses: actions/checkout@v5
+                with:
+                    persist-credentials: false
+
+            -   name: Select and assert Xcode toolchain
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)